packages/expo-application/CHANGELOG.md

```diff
@@ -13,6 +13,7 @@
 ### 💡 Others
 
 - Remove `unimodule.json` in favour of `expo-module.config.json`. ([#25100](https://github.com/expo/expo/pull/25100) by [@reichhartd](https://github.com/reichhartd))
+- On `Android` bump `compileSdkVersion` and `targetSdkVersion` to `34`. ([#24708](https://github.com/expo/expo/pull/24708) by [@alanjhughes](https://github.com/alanjhughes))
```

Comment:
would like to move those changelogs into "breaking changes" section. that would get more attention for the changes.